Sources/Extensions/Array+Truncate.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Array+Truncate.swift
+//  iOSSnapshotTestCase
+//
+//  Created by Jelle Vandebeeck on 28/11/2018.
+//
+
+import Foundation
+
+extension Array {
+    /// Return an array with only the limited amount of elements starting from the rear.
+    ///
+    /// - parameter limit: The total number of items that should be returned.
+    ///
+    /// - returns: The truncated array.
+    public func truncate(by limit: Int) -> Array {
+        return Array(suffix(limit))
+    }
```

Comment:
Moeten we dit niet doen voor de elementen startend vooraan?

Reply:
Goei punt, ik voeg het bij!